extension/Descriptors/AssignmentCategoryDescriptor.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As discussed on slack, I have not been able to get any bulk loading applications to work yet with the extension, so these XML files for descriptors are effectively untested. We'll try to resolve the issues under a new ticket.